DataFrames/Plots/Histogram.js

```diff
@@ -137,6 +139,54 @@ define([
                 win.render();
             };
 
+            /**
+             * Create text for display in info section of the plot
+             * @param {[]} values: list of data values
+             * @returns {String}: info text to display
+             * @private
+             */
+            win._infoText = function(values) {
+
+                var minVal = +1.0e99;
```

Comment:
A better idea is to use the built in constants `Number.MIN_SAFE_INTEGER` and `Number.MAX_SAFE_INTEGER` to define the safe range for which all mathematical operations are meaningful.

The problem for very large integers is that the JavaScript encoding is only unique for every other number. Notice how for equality `Math.pow(2, 53) === Math.pow(2, 53) + 1`